src/core_plugins/tile_map/public/styles/_tilemap.less

```diff
@@ -96,6 +96,8 @@
 
 .leaflet-container {
   background: @tilemap-leaflet-container-bg !important;
+  min-width: 1px !important;//the heatmap layer plugin logs an error to the console when the map is in a 0-sized container
```

Comment:
Could you please insert a space before the comment (or put the comment to another row).